spec/controllers/statistics_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+RSpec.describe StatisticsController, type: :controller do
+  # let(:statistic) { metric_percentage: 100 }
+
+  describe "GET #count_metric for an used metric" do
+    
+    before :each do
+      MetricConfiguration.expects(:all).returns([1, 2, 3])
+      MetricSnapshot.expects(:where).returns([1])
+      get :count_metric, metric_code: 'loc', format: :json
+    end
+
+    it "returns http success" do
+      expect(response).to have_http_status(:success)
```

Comment:
What do you think about using ShouldaMatchers for this kind of assertion? See: http://www.rubydoc.info/github/thoughtbot/shoulda-matchers/Shoulda/Matchers/ActionController#respond_with-instance_method